jina/peapods/remote.py

```diff
@@ -155,9 +158,181 @@ def set_ready(self, resp):
         super().set_ready()
 
 
-class RemoteMutablePod(RemotePea):
-    """A RemoteMutablePod that spawns a remote :class:`MutablePod`.
+def namespace_to_dict(args: Dict) -> Dict:
+    """ helper function to convert argparse.Namespace to json to be uploaded via REST """
+    pea_args = {}
+    for k, v in args.items():
+        if v is None:
+            pea_args[k] = None
+        if isinstance(v, Namespace):
+            pea_args[k] = vars(v)
+        if isinstance(v, list):
+            pea_args[k] = []
+            pea_args[k].extend([vars(_) for _ in v])
+    return pea_args
+
+
+def fetch_files_from_yaml(pea_args: Dict, logger) -> Tuple[set, set]:
```

Comment:
Current expectation is:
- read flow YAML & get all `uses`,  `pymodules` for upload only if they end with `yaml, yml, py`. This is to avoid sending plain text (yaml in string, executor class etc) 
- if `uses` ends with `yaml, yml`, read the files from disk (these are pod yamls)
- check if they also have `pymodules`,  if yes, mark them for upload too.

Since we are reading files only at level 0 & 1, I don't think we need to be recursive (for now).